observatory-platform/observatory/platform/utils/test_utils.py

```diff
@@ -653,13 +654,13 @@ def assert_table_content(self, table_id: str, expected_content: List[dict] = Non
         actual_content = None
         try:
             rows = self.bigquery_client.list_rows(table_id)
-            actual_content = [dict(row) for row in rows]
+            actual_content = json.loads(json.dumps([dict(row) for row in rows], default=str))
         except NotFound:
             pass
 
         self.assertIsNotNone(rows)
         if expected_content is not None:
-            for row in expected_content:
+            for row in json.loads(json.dumps(expected_content, default=str)):
```

Comment:
We shouldn't need to convert the expected items if we have loaded them from JSON or JSON Lines.

Reply:
No problem, I've reverted the expected content in https://github.com/The-Academic-Observatory/observatory-platform/pull/575/commits/b8414f278133ecae1b0a1a8ca8d2f29ad225cdf8